Assets/Scripts/CanonController.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine;
+
+/// <summary>
+/// Controls the canon that shoots the boulders
+/// </summary>
+public class CanonController : MonoBehaviour
+{
+    private Rigidbody2D rb;
+    public GameObject Boulder;
```

Comment:
Rider has some warnings on these variable names- you can standardize them by accepting the suggestion